docs/source/common_issues.rst

```diff
@@ -472,3 +472,41 @@ Here's the above example modified to use ``MYPY``:
 
    def listify(arg: 'bar.BarClass') -> 'List[bar.BarClass]':
        return [arg]
+
+Properties & Incompatible Types in Assignment
+---------------------------------------------
+
+Consider the code below:
+
+.. code-block:: python
+
+    from typing import Union
+    
+    
+    class A:
+        def __init__(self) -> None:
+            self._foo = 0
+    
+        @property
+        def foo(self) -> int:
+            return self._foo
+    
+        @foo.setter
+        def foo(self, value: Union[int, float]) -> None:
+            self._foo = int(value)
+    
+    
+    if __name__ == '__main__':
+        a = A()
+        a.foo = 123
+        a.foo = 456.789
+    
+
+Running Mypy against the above code will generate the following error at ``a.foo = 456.789``: 
+
+.. code-block:: none
+    
+    error: Incompatible types in assignment (expression has type "float", variable has type "int")
+
+While overloading class property setters in Python is valid, Mypy infers the type from the initial ``@property`` definition (e.g., ``int``) as the final type for the attribute, regardless of what the ``@property.setter`` declares (e.g., ``Union[int, float]``). This error may be silenced with `# type: ignore  <http://mypy.readthedocs.io/en/latest/common_issues.html#spurious-errors-and-locally-silencing-the-checker>`_
```

Comment:
Please break this line so each line is under 80 characters, and remove the remark about silencing it with `# type: ignore` -- by the time a user gets here they should know they can silence anything that way.

Also, "mypy" is not capitalized except at the start of a sentence.